### DISCUSSION TITLE
ubuntu-latest-8-cores runners are unavailable

### DISCUSSION THREAD

**Discussion by gaganawhad** [community/community#64104] [Actions]
### Select Topic Area

Bug

### Body

Hi, this was noted here: https://github.com/actions/runner/issues/2742. I am experiencing the same problem at my organization account and wanted to relay it here. Essentially we have 8-core Ubuntu runners that show up as unavailable. Our CI jobs don't seem to be picked up and get caught in an endless loop.

**Comment by gaganawhad** [comment]
Sorry, jobs just got picked up when I created this discussion :-) I am not sure if I misunderstood how these reserved runners work, but I thought they were always available to us. May be this is not really a bug?

**Comment by akhil-capex** [comment]
we are facing this same issue `Waiting for a runner to pick up this job...`

its been more than 30 minutes
